clickhouse-jdbc/src/test/java/ru/yandex/clickhouse/response/ClickHouseResultSetTest.java

```diff
@@ -167,6 +167,7 @@ public void withTotals() throws Exception {
         assertEquals(70511139L, rs.getLong(2));
     }
 
+    @Test
```

Comment:
Nothing changed about this testcase, intellij just warned me it was missing a "@Test" annotation.